src/patternfly/components/Table/images/table--icon-github.hbs

```diff
@@ -0,0 +1,4 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" fill="currentColor" width="30px">
```

Comment:
we don't want to ship these files with the npm package - can you move them to `/static/assets/images/`? (I think that's the dir that serves static images). fwiw, we already have the github and google logos in there. Not sure if they're the same ones, but you may be able to reuse them. We have gitlab, dropdox, and facebook, too.